mycroft/tts/ivona_tts.py

```diff
@@ -0,0 +1,40 @@
+from mycroft.configuration import Configuration
+from mycroft.tts import TTS, TTSValidator
+import pyvona
+
+
+class Ivona(TTS):
+    def __init__(self, lang, config):
+        super(Ivona, self).__init__(lang, config, IvonaValidator(self))
+        self.config = Configuration.get().get("tts", {}).get("ivona", {})
```

Comment:
One last change you'll need is to supply the file extension type by declaring `self.type = '<audio_type>'` [like this](https://github.com/MycroftAI/mycroft-core/blob/b63928888bb81537f011f522dfefc7de1bab5b2c/mycroft/tts/google_tts.py#L24). One thing I noticed is that we only support playing `wav` and `mp3` files so you'll have to make the type `mp3`, also change `ogg` to `mp3` down below, and also change the engines codec with `self.engine.codec = 'mp3'`.

Let me know if this makes sense.

Reply:
Can you see now.